docs/content/bounties/index.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Flow Bug Bounty Program
+description: Earn up to 100k by finding Flow bugs!
+sidebar_title: Bug Bounties
+---
+
+<div>
+  <Button feel="flat" size="large">
```

Comment:
This button doesn't seem to do anything

Reply:
Not hooked up for security reasons. Will connect when approved.